topic/timeseries/time-series-decomposition.ipynb

```diff
@@ -106,12 +106,37 @@
     "engine = sa.create_engine(CONNECTION_STRING, echo=os.environ.get('DEBUG'))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "First, import data into CrateDB. This is a shorthand notation for the same code\n",
+    "illustrated in `timeseries-queries-and-visualization.ipynb`, running corresponding\n",
+    "SQL DDL and DML statements, to load the data."
```

Comment:
@ckurze: Please let us know if you think the SQL-based procedure should go into time-series-decomposition.ipynb instead of timeseries-queries-and-visualization.ipynb.